tests/5.0/simd/test_simd_nontemporal.c

```diff
@@ -0,0 +1,46 @@
+//===--- test_simd_nontemporal.c ------------------------------------------------------------------------===//
+//
+// OpenMP API Version 5.0 Nov 2018
+//  
+// This test checks for support of the nontemporal clause on a simd construct. The nontemporal clause 
+// indicates that accesses to the storage location of list items have low temporal locality across the
+// iterations in which those storage locations are accessed. 
+//  
+////===--------------------------------------------------------------------------------------------------===//
+
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+
+#define N 1028
+
+int test_simd_nontemporal() {
+   int errors = 0;
+   int i;
+   int a[N], b[N], c[N];
+
+   for (i = 0; i < N; i++) {
+      a[i] = 10; 
+      b[i] = i;
+      c[i] = 2 * i;
+   }   
+
+   #pragma simd nontemporal (a, b, c)
```

Comment:
When you say nontemporal,  you are telling the compiler that the accesses have low temporal locality...increment "i" differently so that you are not accessing too many elements in a single cache line.

Reply:
@spophale made some changes, let me know if this is o.k. and then i'll go ahead and push the device version of this code.